src/docs/asciidoc/reference/configuration/account-configuration.adoc

```diff
@@ -35,13 +39,6 @@ include::{docdir}/common/definitions/app-name.adoc[]
 | appName
 | --app-name <name>
 
-| Environment name
-| 
-include::{docdir}/common/definitions/env-name.adoc[]
-| ZP_ENV_NAME
-| envName
-| --env-name <env>
-
 | Platform URL
 | 
 include::{docdir}/common/definitions/platform-url.adoc[]
```

Comment:
By default, the developer uses the ~production environment of ZetaPush~ **public ZetaPush cloud** to develop and deploy its applications. In most cases, the developer doesn’t need to set this property. This property if useful when another (on premise, or dedicated) ZetaPush ~platform~ **cloud** is to be used.